e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation-appeal-site.feature

```diff
@@ -0,0 +1,59 @@
+@wip
+@UI-ONLY
+Feature: Appeal Submission Save and Continue Navigation - About the appeal site
+As a prospective appellant, I want to be taken through the submission process efficiently
+so that I do not get confused.
+This feature file covers the navigation between the About-the-appeal-site to the task-list
+The navigation does not depend on the status of a sub-section
+
+Scenario: Navigation - About the appeal site
+  Given the "Site location" is presented
+  When the "Site location" is submitted with valid values
+  Then the "Site ownership" is presented
+
+Scenario: Navigation - Site ownership with YES selected
+  Given the "Site ownership" is presented
+  When the "Site ownership" is submitted with a YES valuegit
```

Comment:
I think there is a typo at valuegit

Reply:
I was typing in the terminal and it ended up there, didn't notice, thank you